website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -49,6 +49,54 @@ You can read more about each of these behavior changes in the following links:
 
 - (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/resource-properties/arguments#supported-types).
 
+### Deprecation warnings
+
+New deprecation warnings in v1.10.
+
+#### Duplicate keys in profiles.yml
+
+Previously, if identical keys existed in the [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml), dbt would use the last configuration listed in the file. 
+
+```yml
+
+my_profile:
+  target: 
+  outputs:
+...
+
+my_profile: # dbt would use this profile key
+  target: 
+  outputs:
+...
+
+```
+
+dbt will stop supporting duplicate keys with silent overwrite in the future. In v1.10, if two identical keys exist in the `profiles.yml`, you will get a warning, and in a future version, dbt will stop supporting this entirely. 
+
+#### Custom inputs
+
+Historically, dbt has allowed you to configure inputs largely unconstrained. A common example of this is setting custom YAML properties:
+
+```yml
+
+models:
+  - name: my_model
+    description: A model in my project.
+    dbt_is_awesome: true # a custom property
+
+```
+
+dbt sees that you've configured an unrecognized custom property (`dbt_is_awesome`) and silently continues. Without a set of strictly defined inputs, it becomes much more challenging to determine if a configuration in your project is valid. This creates unintended issues such as:
+- Silently ignoring misspelled properties and configurations (for example, `desciption:` instead of `description:`).
+- Unintended collisions with user code when dbt introduces a new “reserved” property or configuration.
+
+Starting in dbt v1.10, you will receive deprecation warnings for dbt code that will become invalid in the future, including: 
```

Comment:
Same comment as above. I recommend moving the deprecation warning up and providing the historical context if they want to read more.